cli/server/server.go

```diff
@@ -405,7 +405,7 @@ func mkOracle(config network.ServerConfig, chain *core.Blockchain, serv *network
 	return orc, nil
 }
 
-func mkConsensus(config network.ServerConfig, chain *core.Blockchain, serv *network.Server, log *zap.Logger) (consensus.Service, error) {
+func mkConsensus(config network.ServerConfig, chain *core.Blockchain, serv *network.Server, log *zap.Logger, restartCh chan struct{}) (consensus.Service, error) {
```

Comment:
@roman-khimov, not sure whether we need a channel for consensus restart. We may go with callback if needed.

Also, current PR logic allows to move consensus reloading completely outside of consensus server code.